tmt/export/__init__.py

```diff
@@ -40,7 +40,11 @@
 if TYPE_CHECKING:
     import tmt.base
 
-TEMPLATES_DIRECTORY = Path(pkg_resources.resource_filename('tmt', 'export/templates'))
+TEMPLATES_DIRECTORY = (Path(importlib.resources.files('tmt'))  # type: ignore[arg-type]
```

Comment:
So we need this five times, the type ignore copy pasting is not very nice. What about introducing a `tmt.utils.locate()` function which would take care of this? Then it could look like this:

```suggestion
TEMPLATES_DIRECTORY = tmt.utils.locate("export/templates")
```

Not sure about the name, perhaps something more specific like `locate_path()`? I can't think of anything better right now. @happz, any thoughts here?

Reply:
Yeah, it seems to be pretty common. I'm not sure a helper would be always available, there might be cyclic imports lurking around, but I for one would give it a try.

Reply:
`importlib_resource_files()`? Both `locate()` and `locate_path()` lack the context, and this would be a very limited use case, "find me a path to a file bundled into tmt package"

Reply:
Sounds good.

Reply:
Yes, nice, thanks!